signing/signing.go

```diff
@@ -0,0 +1,50 @@
+package signing
+
+import (
+	"context"
+	"log"
+	"net/http"
+	"path/filepath"
+	"strings"
+
+	"github.com/gorilla/mux"
+	"github.com/ossf/scorecard/v2/cron/data"
+)
+
+type ScorecardOutput struct {
+	JsonOutput string
+}
+
+func GetResults(w http.ResponseWriter, r *http.Request) {
+	// Get params to build GCS filepath.
```

Comment:
Using `http.Error` + return seems really prone to mistakes (it's the same way C developers always forget to free arrays). I know that's how the API works, but I think it may be better to do something like:
```
func GetResults(w http.ResponseWriter, r *http.Request) {
results, err := getResults()
if err != nil{
   http.Error(w, err)
   return
}

  w.Header().Set("Content-Type", "application/json")
  w.Write(resultsBytes)
}
```

Reply:
I refactored the code to align with this